src/hooks/useTodosState.ts

```diff
@@ -13,21 +14,238 @@ export const useTodosState = (initialTodos?: Array<Todo>): {
     handleTodoCompleted: (id: Todo["id"], completed: Todo["completed"]) => void
     handleChangeTodoTitle: (id: Todo["id"], title: Todo["title"]) => void
 } => {
-    const [todos, setTodos] = useState(initialTodos ?? []);
+    const [todos, setTodos] = useState([] as Array<Todo>);
+    const [areLoaded, setAreLoaded] = useState(false);
+    const [activeCount, setActiveCount] = useState(0);
```

Comment:
activeCount y completedCount son estados dependientes del estado todos, con lo cual pierden el sentido, siempre que actualices todos debes si o si actualizar estos otros dos para mantener consistencia, haciéndolos inútiles